doc/reverse_engineering/game_mechanics/swgb/shields.md

```diff
@@ -0,0 +1,51 @@
+# Shields mechanics in SWGB
+
+## Intro
+
+Shields are another unique feature of the SWGB franchise. A shielded unit has
+double the number of hit points that of an unshielded one, and those extra hit
+points regenerate.
+
+## Mechanism
+
+A shielded unit gradually fills a shield bar (originally yellow and placed
+under the health bar), that doubles its health points. Aditionally, shielded
+units have a yellow glow effect, and the UI lights the left indicator in yellow
+when a shielded unit is selected, as visible
+[here]([200~https://www.youtube.com/watch?v=pwAtMv_eiM4&feature=youtu.be&t=1786]).
+
+Shields are depleted first when taking hits, before the regular health bar.
+Regenerating shields do not affect the health bar.
+
+### Modifiers
+
+When losing the shield ability (like when exiting a shield generator), the
+shield points will gradually deplete. They will gradually replenish when
```

Comment:
Is the depletion rate fixed?

Reply:
No idea, it would have to be measured, I unfortunately don't have a running instance of the game right now.

See in the MR body:

> Notable missing stuff is shield regen/losses rates, but it could be in the converted assets files.

Reply:
I looked it up in the .dat and it is apparently a fixed time of 40s. Have not tested it though.